.gitmodules

```diff
@@ -24,7 +24,8 @@
 	url = https://github.com/rust-lang-nursery/rustfmt.git
 [submodule "src/tools/miri"]
 	path = src/tools/miri
-	url = https://github.com/rust-lang/miri.git
+	url = https://github.com/gnzlbg/miri.git
+	branch = is_const_eval
```

Comment:
You shouldn't commit this. The submodule update should suffice

Reply:
That commit should be removed anyways once the changes land on `miri`.

Reply:
The submodule update updates miri to my private branch, so I don't think that can be done here either.